visual_relation/visual_relation_tutorial.ipynb

```diff
@@ -430,7 +430,7 @@
        "lf_carry_subject          0   1.000000  \n",
        "lf_not_person             3   0.625000  \n",
        "lf_ydist                  8   0.466667  \n",
-       "lf_dist                   6   0.500000  \n",
+       "lf_dist                  13   0.500000  \n",
```

Comment:
why do we think this count changed?

Reply:
It could be that I ran this on the sample dataset (downloaded for faster tests) instead of the full one..

Reply:
also looks like I have python3.7 instead of 3.6

Reply:
also chance this was never re-run with the latest version of snorkel, esp after this change: https://github.com/snorkel-team/snorkel-tutorials/commit/b0de3a6f0257628e1fcfd91a0d8f12c6af47ab24

we likely need a broader solution for making sure _all_ the notebooks are re-run and re-committed after a diff that affects more than 1 tutorial (e.g. a version change)